### PR TITLE
Get the Actions bot out of the PR review business.

### PR DESCRIPTION
**Pull request by BillyONeal**

### CODE REVIEW

**Review by ras0219-msft — DISMISSED** [review]
This LGTM

**Comment by BillyONeal** [comment]
I cleaned up the markdown output, here's what an error looks like:

image

**Comment by BillyONeal** [comment]
With only warnings it just makes a green check, so the user needs to go find it:

image

image

**Review by JonLiu1993 — DISMISSED** [review]

**Comment by autoantwort** [comment]
Hm since you get a green ci I guess that most users will never detect the warnings without a comment (You have to know that there could be warnings that you are searching for, but how are new users supposed to know that).

File: .github/workflows/untrustedPR.yml [hunk 1]
```diff
@@ -64,84 +57,78 @@ jobs:
...
+                license_output += missing_license
+                license_output += "\n\nValid values for the license field can be found at https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#license\n\n"
               }
               if (deprecated_license !== "") {
-                output += 'If you feel able to do so, please consider replacing the deprecated license identifiers in the following files:\n'
-                output += deprecated_license
-                output += "\n\nDeprecated and non deprecated license identifiers can be found [here](https://spdx.github.io/spdx-spec/SPDX-license-list/#a3-deprecated-licenses)\n"
+                license_output += 'If you feel able to do so, please consider replacing the deprecated license identifiers in the following files:\n'
+                license_output += deprecated_license
+                license_output += "\n\nDeprecated and non deprecated license identifiers can be found at https://spdx.github.io/spdx-spec/SPDX-license-list/#a3-deprecated-licenses\n"
```
Comment (autoantwort) [inline .github/workflows/untrustedPR.yml:126]:
```suggestion
                license_output += "\n\nDeprecated and non deprecated license identifiers can be found at https://spdx.org/licenses/#deprecated\n"
The old link is now 404
```

File: .github/workflows/untrustedPR.yml [hunk 2]
```diff
@@ -64,84 +57,79 @@ jobs:
...
+                license_output += "\n\nValid values for the license field can be found at https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#license\n\n"
               }
               if (deprecated_license !== "") {
-                output += 'If you feel able to do so, please consider replacing the deprecated license identifiers in the following files:\n'
-                output += deprecated_license
-                output += "\n\nDeprecated and non deprecated license identifiers can be found [here](https://spdx.github.io/spdx-spec/SPDX-license-list/#a3-deprecated-licenses)\n"
+                license_output += 'If you feel able to do so, please consider replacing the deprecated license identifiers in the following files:\n'
+                license_output += deprecated_license
+                license_output += "\n\nDeprecated and non deprecated license identifiers can be found at https://spdx.org/licenses/#deprecated\n"
+The old link is now 404
```
Comment (autoantwort) [inline .github/workflows/untrustedPR.yml:127]: Whut? You merged a broken version of the workflow :)

**Pull request merged**